FHIR_README.md

```diff
@@ -0,0 +1,77 @@
+# OpenEMR FHIR API Documentation
+
+### Overview
+
+Easy-to-use JSON-based REST API for OpenEMR FHIR. All code is done in classes and separate from the view to help with codebase modernization efforts. See standard OpenEMR API docs [here](API_README.md)
+
+### Prerequisite
+Enable this FHIR service in OpenEMR menu: Administration->Globals->Connectors->"Enable FHIR Provider Client Service" 
```

Comment:
The global to turn this on is the same as the API (there is no FHIR specific global), so should be:
```
Enable this API service in OpenEMR menu: Administration->Globals->Connectors->"Enable OpenEMR REST API"
```

Reply:
Hello @bradymiller  I tried making calls to FHIR with just API turned on and it failed. Until I selected HAPI server, as against default or so, I will check to confirm this again

Reply:
hi @prondubuisi , That should not be the case. Rec. doing a search for `fhir_enable` in codebase to see where the "Enable OpenEMR REST API" is used (spoiler alert; just controls a couple menu items :) ).

Reply:
Alas! I am wrong again. making the update.

Reply:
no prob and it illustrates a good point regarding that FHIR global; that it is likely to cause confusion to end users in the future :)

Reply:
Yeah, thought about it. Since we want Independent API and FHIR services how about we have independent globals, what do you think @bradymiller